src/components/projectSelect/index.tsx

```diff
@@ -62,18 +90,27 @@ export const ProjectSelect: React.FC<ProjectSelectProps> = ({
   return (
     <SearchableDropdown
       label="Project"
-      value={selectedProject?.displayName || selectedProject?.identifier}
+      value={
+        selectedProject?.displayName ||
+        selectedProject?.identifier ||
+        selectedProjectIdentifier
```

Comment:
Why are these 3 necessary? i would imagine the original code is enough. 
if `selectedProjectIdentifier` does not exist we shouldn't be showing it at all.

Reply:
It's necessary for repos. We set selectedProjectIdentifier to be the repo display name.